### DISCUSSION TITLE
How to build, install and deploy extensions to Digital Ocean

### DISCUSSION THREAD

**Discussion by dpihac** [directus/directus#15700] [Archive]
Hi! I've moved from Docker deployment on Digital Ocean to GitHub because of the extensions. Now, I have created test extension which works nicely locally, but I would like to automatize the process.

Extension is in the `src/project-name/extension-name` directory. When I need to install it manually, I follow instructions to copy it to the `src/project-name/extensions/endpoint/extension-name` directory.

Question is:

- how to build multiple extensions during deployment and
- copy them to the `extensions` directory (there might be different types of extensions)

I don't want to publish them to the npm. I could probably write some scripts, but I thought there was another way. Another solution would be to create a custom Docker image.

Any other ideas? Thanks 🙏

**Comment by rijkvanzanten** [comment]
Writing some custom scripts to build and put them in the correct location sounds like the way to go to me 👍🏻
